src/memory_test.cpp

```diff
@@ -22,14 +22,11 @@ class AllocatorTest : public testing::Test {
 };
 
 TEST_F(AllocatorTest, AlignedAllocate) {
```

Comment:
This looks like a perfect use for a parametrized test!

Reply:
Ack. I'll see if I can clean this up in another pass.